src/Razor/test/Microsoft.AspNetCore.Razor.LanguageServer.Test/Formatting_NetFx/TestRazorFormattingService.cs

```diff
@@ -43,6 +44,15 @@ public static async Task<IRazorFormattingService> CreateWithFullSupportAsync(
 
         var hostServicesProvider = new DefaultHostServicesProvider();
 
-        return new RazorFormattingService(mappingService, hostServicesProvider, languageServerFeatureOptions, loggerFactory);
+        var service = new RazorFormattingService(mappingService, hostServicesProvider, languageServerFeatureOptions, loggerFactory);
+        foreach (var validation in service.GetTestAccessor().FormattingValidationPasses)
+        {
+            if (validation is FormattingContentValidationPass contentValidationPass)
```

Comment:
This was a bit hacky but I don't currently know of a better option

Reply:
I think it would be a bit nicer to put a method on the test accessor that does this. so `server.GetTestAccessor().SetDebugAsserttsEnabled(false)`, then the hacky bit is at least in the service that owns the data that we're hacking in to.